src/createTokenBridge.integration.test.ts

```diff
@@ -161,6 +160,46 @@ it(`successfully deploys token bridge contracts through token bridge creator`, a
   expect(tokenBridgeContracts.orbitChainContracts.beaconProxyFactory).not.toEqual(zeroAddress);
   expect(tokenBridgeContracts.orbitChainContracts.upgradeExecutor).not.toEqual(zeroAddress);
   expect(tokenBridgeContracts.orbitChainContracts.multicall).not.toEqual(zeroAddress);
+
+  // set weth gateway
+  const setWethGatewayTxRequest = await createTokenBridgePrepareSetWethGatewayTransactionRequest({
+    rollup: testnodeInformation.rollup,
+    parentChainPublicClient: nitroTestnodeL1Client,
+    orbitChainPublicClient: nitroTestnodeL2Client,
+    account: l2RollupOwner.address,
+    retryableGasOverrides: {
+      gasLimit: {
+        base: 100_000n,
+      },
+    },
+    tokenBridgeCreatorAddressOverride: tokenBridgeCreator,
+  });
+
+  // sign and send the transaction
+  const setWethGatewayTxHash = await nitroTestnodeL1Client.sendRawTransaction({
+    serializedTransaction: await l2RollupOwner.signTransaction(setWethGatewayTxRequest),
+  });
+
+  // get the transaction receipt after waiting for the transaction to complete
+  const setWethGatewayTxReceipt = createTokenBridgePrepareSetWethGatewayTransactionReceipt(
+    await nitroTestnodeL1Client.waitForTransactionReceipt({ hash: setWethGatewayTxHash }),
+  );
+
+  // checking retryables execution
+  const orbitChainSetGatewayRetryableReceipt = await setWethGatewayTxReceipt.waitForRetryables({
+    orbitPublicClient: nitroTestnodeL2Client,
+  });
+  expect(orbitChainSetGatewayRetryableReceipt).toHaveLength(1);
+  expect(orbitChainSetGatewayRetryableReceipt[0].status).toEqual('success');
+
+  // verify weth gateway
+  const registeredWethGateway = await nitroTestnodeL1Client.readContract({
```

Comment:
Instead of doing `nitroTestnodeL1Client.readContract`, let's instead refetch the contracts through the create token bridge tx receipt: `txReceipt.getTokenBridgeContracts`?